test/javascripts/unit/admin/modules/breadcrumb_preview_test.js

```diff
@@ -0,0 +1,23 @@
+module("BreadcrumbPreview", function(){
```

Comment:
The intent of this test is indeed to be minimal. We believe that testing the other methods wasn't beneficial as both would depend on building a fixture for them to be tested. I'm not keen on fixture testing as the structure of the DOM can change. It's likely that the next developer touching this code doesn't know about it. The fixture can still be a passing fixture even thought is not representing the real thing.